assets/src/stories-editor/components/custom-video-block-edit.js

```diff
@@ -188,6 +188,7 @@ class CustomVideoBlockEdit extends Component {
 			// selected media, then switches off the editing UI
 			setAttributes( { src: media.url, id: media.id } );
 			this.setState( { src: media.url, editing: false, duration: null, videoSize: null } );
+			maybeSupplyPoster.call( this, media );
```

Comment:
We're already providing the poster image in `componentDidUpdate` if the video has a featured image (`videoFeaturedImage`). So what's the difference to that approach?

Reply:
Good point, that's already provided.